sdk/cosmosdb/cosmos/test/public/functional/item/item.spec.ts

```diff
@@ -550,6 +552,53 @@ describe("Create, Upsert, Read, Update, Replace, Delete Operations on Item", fun
       });
       assert.strictEqual(resource1.id, resource2.id);
     });
+    it("Test diagnostics for item CRUD", async function () {
+      const container = await getTestContainer("db1", undefined, { partitionKey: "/id" });
+      const createTimestamp = getCurrentTimestampInMs();
+      const itemId = "2";
+      const { diagnostics: createDiagnostics } = await container.items.create({ id: itemId });
+      validateDiagnostics(createDiagnostics, {
+        requestStartTimeUTCInMsLowerLimit: createTimestamp,
+        requestEndTimeUTCInMsHigherLimit: getCurrentTimestampInMs(),
+        retryCount: 0,
+        metadataCallCount: 2, // One call for database account + data query call.
```

Comment:
@v1k1 should we consider storing actual endpoints or maybe just the route of the call instead of storing counts?